.cicdtemplate/.codemagic/README.md

```diff
@@ -0,0 +1,34 @@
+# Setup
+
+Setting up the [Codemagic](https://codemagic.io/start/) to the project starts from adding project to
+the Codemagic console.
+After that, you can follow the instruction below to setup the environment variables and deployment.
+
+# Environment Variables
+
+You can configure the environment variables for Codemagic by accessing the `Environment Variables`
+tab under `App Settings`.
+For detailed instructions on how to configure these variables, please refer
+to [this document](https://docs.codemagic.io/yaml-basic-configuration/configuring-environment-variables/#configuring-environment-variables)
+.
+
+# Deploy to Firebase App Distribution (FAD)
+
+1. Set up the Firebase SDK in your app by following
+   the [Get started guide for Android](https://firebase.google.com/docs/android/setup).
+2. Create service account for Codemagic by following
+   the [Firebase App Distribution with codemagic.yaml](https://docs.codemagic.io/yaml-publishing/firebase-app-distribution/)
+   at `Authenticating via service account` which is recommended.
+3. Update the script on `app_id` and tester `groups` in `codemagic.yaml` file:
+
+```yaml
+  publishing:
+  firebase:
+     firebase_service_account: $FIREBASE_SERVICE_ACCOUNT
```

Comment:
is `$FIREBASE_SERVICE_ACCOUNT` a JSON file content? Please clarify

Reply:
@luongvo It's from setting up the Firebase Service Account. You can see the [document here](https://docs.codemagic.io/yaml-publishing/firebase-app-distribution/).